Assets/MRTK/Examples/Experimental/ScrollingObjectCollection/Scripts/ScrollableListPopulator.cs

```diff
@@ -15,23 +16,25 @@ public class ScrollableListPopulator : MonoBehaviour
     {
         [SerializeField]
         [Tooltip("The ScrollingObjectCollection to populate, if left empty. the populator will create on your behalf.")]
-        private ScrollingObjectCollection scrollCollection;
+        private ScrollingObjectCollection scrollView;
+
+        private GridObjectCollection gridObjectCollection;
```

Comment:
>private GridObjectCollection gridObjectCollection; [](start = 8, length = 50)

should this also be configurable from outside? (like scrollView?)
also I would move it further down as it's between the scrollView property / serialized field declaration

Reply:
Moving field. This class was made as an example of  how to configure the scroll view. So grid is only copying values used on scroll view definition.